### PR TITLE
Export as default SellerContext to use in other Apps with imports

### PR DESCRIPTION
**Pull request by gustavopvasconcellos**
#### What does this PR do? \*
Export as default SellerContext to use in other App's imports like:

```
import { SellerContext } from 'vtex.seller-selector'
```

### CODE REVIEW

File: react/CurrentSellerContext.tsx [hunk 1]
```diff
@@ -30,3 +30,5 @@ export const CurrentSellerProvider: FC<ProviderProps> = ({
     </CurrentSellerContext.Provider>
   )
 }
+
+export default CurrentSellerContext
```
Comment (klzns) [inline react/CurrentSellerContext.tsx]: Why is that necessary? Why not just import the CurrentSellerPovider?

You can import it as:

```
import { CurrentSellerProvider } from 'vtex.seller-selector/CurrentSellerContext'
```

or

```
import { useCurrentSeller } from 'vtex.seller-selector/CurrentSellerContext'
```

**Review by klzns — CHANGES_REQUESTED** [review]

File: react/CurrentSellerContext.tsx (hunk 1 above)
Comment (gustavopvasconcellos) [reply react/CurrentSellerContext.tsx]: @klzns Sorry I didn't try this method to import the contexts, its worked fine here! Thank you!

Comment (klzns) [reply react/CurrentSellerContext.tsx]: No problem! ✌️

Comment (gustavopvasconcellos) [reply react/CurrentSellerContext.tsx]: @klzns Actually... I used this way to import the Contexts but isn't working, just `vtex link` pass with success, but the component on front gives me this error:

image

Code example:
image

Comment (gustavopvasconcellos) [reply react/CurrentSellerContext.tsx]: @klzns some news about that?

Comment (klzns) [reply react/CurrentSellerContext.tsx]: Sorry, I'll get to you in a second

**Review by klzns — APPROVED** [review]

**Comment by klzns** [comment]
Now this will work:

`import { useCurrentSeller } from 'vtex.seller-selector/CurrentSellerContext'`

🚀
